src/macaron/slsa_analyzer/checks/provenance_available_check.py

```diff
@@ -174,8 +174,8 @@ def find_provenance_assets_on_package_registries(
                     # adversary provides a super large malicious file.
 
                     # TODO: refactor the size checking in this check and the `provenance_l3_check`
-                    # so that we have consistent behavior when checking provenance size.
-                    # The schema of the ini config also needs changing.
+                    #  so that we have consistent behavior when checking provenance size.
+                    #  The schema of the ini config also needs changing.
```

Comment:
Nit: Please revert this change. I know it is not that important, but since it messes with `git blame` I figure we should not keep it.